tests/cargo-kani/foreign-function-stubbing/src/main.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+//! This tests stubbing for foreign functions and methods.
+
+use other_crate;
+
+fn the_answer() -> u32 {
+    42
+}
+
+// TODO: Split up these assertions into separate harnesses, once stubbing is able to support that.
```

Comment:
What about at least splitting this test into multiple ones? I think it will make it much more readable.

Reply:
Done - split into public functions, private functions, and methods.